src/AgConfigure.cpp

```diff
@@ -81,90 +98,55 @@ void Configuration::saveConfig(void) {
 
 void Configuration::loadConfig(void) {
   bool readSuccess = false;
+  char *buf = (char *)malloc(EEPROM_CONFIG_SIZE);
+  if (buf == NULL) {
+    logError("Malloc read file buffer failed");
+    return;
+  }
+  memset(buf, 0, EEPROM_CONFIG_SIZE);
 #ifdef ESP8266
-  uint8_t *data = (uint8_t *)&config;
-  for (int i = 0; i < sizeof(config); i++) {
-    data[i] = EEPROM.read(i);
+  for (int i = 0; i < EEPROM_CONFIG_SIZE; i++) {
+    buf[i] = EEPROM.read(i);
   }
   readSuccess = true;
 #else
   File file = SPIFFS.open(CONFIG_FILE_NAME);
   if (file && !file.isDirectory()) {
-    if (file.readBytes((char *)&config, sizeof(config)) == sizeof(config)) {
-      readSuccess = true;
-    }
+    logInfo("Read file");
```

Comment:
Since we are  about to read the file I propose to change the log message to
"Reading file"

Reply:
Ok